newapi/ooniapi/measurements.py

```diff
@@ -289,13 +291,18 @@ def _fetch_autoclaved_measurement_body(report_id: str, input) -> dict:
 def _fetch_measurement_body(report_id, input: str) -> bytes:
     """Fetch measurement body from either disk, jsonl or autoclaved on S3"""
     log.debug(f"Fetching body for {report_id} {input}")
-    if report_id.count("_") == 5:
+    u_count = report_id.count("_")
+    if u_count == 5:
         # Look on disk and then from JSONL cans on S3
         body = _fetch_measurement_body_on_disk(report_id, input)
         if body is None:
             log.debug(f"Fetching body for {report_id} {input} from jsonl on S3")
             body = _fetch_jsonl_measurement_body(report_id, input)
-    elif report_id.count("_") == 2:
+    elif u_count == 2:
+        body = _fetch_autoclaved_measurement_body(report_id, input)
+    elif u_count == 1:
```

Comment:
Are we sure the underscores are a good indicator of which client generated the test? If so we should probably document in here what the mappings of underscore count to probe_version looks like.

Reply:
Added comment